src/CommonTreeDropdown.php

```diff
@@ -133,11 +133,10 @@ public function adaptTreeFieldsFromUpdateOrAdd($input)
        // leading/ending space will break findID/import
         $input['name'] = trim($input['name']);
 
-        if (
-            isset($input[$this->getForeignKeyField()])
-            && !$this->isNewID($input[$this->getForeignKeyField()])
-            && $parent->getFromDB($input[$this->getForeignKeyField()])
-        ) {
+        $fk = static::getForeignKeyField();
```

Comment:
Method is not static
```suggestion
        $fk = $this->getForeignKeyField();
```

Reply:
it is static, see https://github.com/glpi-project/glpi/blob/8704313222851b3185e3a96b11e8b30bdcbfbadd/src/CommonDBTM.php#L246

Reply:
Oh, indeed, sorry